Ash.FrontEnd/src/events/EventsCalendar/style.less

```diff
@@ -0,0 +1,118 @@
+.calendar {
+  display: block;
+  position: relative;
+  width: 100%;
+  background: @app-background-color;
+  border: 1px solid @cal-border-color;
+
+  .header {
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 115%;
+    padding: 1.5em 0%;
+    border-bottom: 1px solid @cal-border-color;
+  }
+  .days-of-week {
+    text-transform: uppercase;
+    font-weight: 400;
+    color: @cal-text-color-light;
+    font-size: 70%;
+    padding: 0.75em 0%;
+    border-bottom: 1px solid @cal-border-color;
+  }
+  .body {
+    .cell {
+      position: relative;
+      height: 5em;
+      border-right: 1px solid @cal-border-color;
+      overflow: hidden;
+      cursor: pointer;
+      background: @app-background-color;
+      transition: 0.25s ease-out;
+
+      :hover {
+        background: @app-background-color;
+        transition: 0.5s ease-out;
+
+        .background {
+          opacity: 0.2;
+          transition: 0.5s ease-in;
+          color: lightcoral;
+        }
+      }
+      .number {
+        position: absolute;
+        font-size: 82.5%;
+        line-height: 1;
+        top: 0.75em;
+        right: 0.75em;
+        font-weight: 700;
+      }
+      .disabled {
+        color: @cal-text-color-light;
+        pointer-events: none;
+      }
+      .background {
+        font-weight: 700;
+        line-height: 1;
+        color: @cal-main-color;
+        opacity: 0;
+        font-size: 7em;
+        position: absolute;
+        top: -0.05em;
+        right: -0.1em;
+        transition: 0.25s ease-out;
+        letter-spacing: -0.07em;
+      }
+    }
+    .row {
+      border-bottom: 1px solid @cal-border-color;
+      :last-child {
+        border-bottom: none;
+      }
+    }
+    .selected {
+      .background {
+        opacity: 0.4;
+        transition: 0.5s ease-in;
+      }
+    }
+    .column {
+      flex-grow: 1;
+      flex-basis: calc(98% / 7);
+      width: calc(98% / 7);
+      &:last-child {
+        border-right: none;
+      }
+    }
+  }
+}
+/* GRID */
+.row {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+}

```

Comment:
I would be curious to see if this could be done with CSS Grids instead of Flexboxes.